script/volumesnapshotlocation.yaml

```diff
@@ -26,3 +26,4 @@ spec:
     region: minio
     s3Url: MINIO_ENDPOINT
     s3ForcePathStyle: "true"
+    restoreAllIncrementalSnapshots: "true"
```

Comment:
fullRestore?

Reply:
fullRestore may confuse with restore of full backup.